testcase.py

```diff
@@ -312,11 +318,11 @@ def initialize(self, start_time, warmup_period, end_time=np.inf):
 
         Parameters
         ----------
-        start_time: int
+        start_time: float
             Start time of simulation to initialize to in seconds.
-        warmup_period: int
+        warmup_period: float
```

Comment:
@SenHuang19 A reason to change warmup_period and start_time to float instead of int?  I think it may be safer to use int?